app-config.yaml

```diff
@@ -42,13 +42,15 @@ backend:
 integrations:
   github:
     - host: github.com
-      apps:
```

Comment:
You have to revert this change as the integration is the only way we are supporting this moving forward.